.circleci/ci/src/workflows/workflow-nexus-staging.ts

```diff
@@ -0,0 +1,30 @@
+import { SlackAnnouncementJob, SetupJob, NexusStagingJob } from '../jobs';
+import { Config, Workflow, workflow } from '@circleci/circleci-config-sdk';
+import { config } from '../config';
+import { CircleCIEnvironment } from '../pipelines';
+
+export class WorkflowNexusStaging {
+  static create(dynamicConfig: Config, environment: CircleCIEnvironment) {
+    const setupJob = SetupJob.create(dynamicConfig);
+    dynamicConfig.addJob(setupJob);
+
+    const nexusStagingJob = NexusStagingJob.create(dynamicConfig, environment);
+    dynamicConfig.addJob(nexusStagingJob);
+
+    const slackAnnouncementMessage = `🎆 APIM - ${environment.graviteeioVersion} released!`;
```

Comment:
if it's used only in the job maybe it can be a const in the job no ?

Reply:
The job is reused by other workflows with different messages. So for me the message definition needs to be at the workflow level.